### PR TITLE
Endpoint view

### PR DESCRIPTION
**Pull request by alovegren**
Addresses issue #16; created `onNewEndpointClick` event handler and bound to 'new bin' click, along with supporting html and basic mvc circuitry in `controller.js`. 

My HTML semantics are garbage but that can be revisited especially whilst styling 🙃

### CODE REVIEW

File: public/javascripts/controller.js [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+  }
+
+  #bindEvents() {
+    const newBinBtn = document.getElementById('newBinBtn');
+    newBinBtn.onclick = this.addEndpoint.bind(this);
+  }
+  
+  constructor(model, view) {
+    this.model = model;
+    this.view = view;
```
Comment (andrewdmoore84) [inline public/javascripts/controller.js:16]: Can we make `model` and `view` private variables as well? (hashnames?)

**Review by andrewdmoore84 — CHANGES_REQUESTED** [review]
Hope it's ok to request a change, just thinking we should encapsulate model and view as well by making them private hashname variables

**Comment by Jooliashi** [comment]
Feel bad that model is not done yet so missy you can't see your code visually! Something i just noticed we might need to change is to add another handlebar javascript `src` tag and nav handlebar in the `endpoint.html` as I'm trying out `localhost:3000/hi`(`hi` is interpreted as a endpoint address). because i don't think the code from view can reference both html at the same time. Unless we would just not have the nav bar in `endpoint` view? Since I also notice you used `afterbegin` and in that case, the endpoint view will appear above nav if nav exists? Just wanna verify your design intentions and don't accidentally override anything XD

**Comment by alovegren** [comment]
No need to feel bad! I could have started with `model` if I wasn't OK kind of imagining some methods.

And yes, good call with the `afterbegin` -- so used to just having the one main template to plop in there! I went ahead and hard-coded the same template into `endpoint.html` but we can refactor that to be less repetitive if we have time.

**Comment by alovegren** [comment]
I also went ahead and added some conditional logic to make sure that the 'user clicks new potato bin' event is bound only for `index.html`

**Review by andrewdmoore84 — APPROVED** [review]

**Pull request merged**